CHANGES

```diff
@@ -41,12 +41,12 @@ By: @scttcper (#50456)
 
 ### Escalating Issues V2 (EA) (ongoing)
 
-This is a placeholder milestone for tracking post-v1 work for escalating issues.
-See the RFC: https://github.com/getsentry/rfcs/blob/main/text/0078-escalating-issues.md
-There's also discussions about escalation in here:
-https://github.com/getsentry/sentry/discussions/43039
-
-V2 will focus on moving ongoing issues (upcoming) into the state archived-until-escalating after an issue being in the `ongoing` state for 14 days.
+This is a placeholder milestone for tracking post-v1 work for escalating issues.
```

Comment:
There's no GitHub in this line of text. Unsure what caused a diff to be generated.